wallet/chains/pchain/builder.go

```diff
@@ -71,6 +55,69 @@ func buildCreateSubnetTx(wallet *primary.Wallet, params *pchainTxs.CreateSubnetT
 	return types.BuildTxResult{BuildTxOutput: pChainResult}, nil
 }
 
+// buildCreateChainTx provides a default implementation that can be used by any wallet
+func buildCreateChainTx(wallet *primary.Wallet, account account.Account, params *pchainTxs.CreateChainTxParams) (types.BuildTxResult, error) {
+	subnetAuthKeys, err := convertSubnetAuthKeys(params.SubnetAuthKeys)
+	if err != nil {
+		return types.BuildTxResult{}, fmt.Errorf("failed to convert subnet auth keys: %w", err)
+	}
+	options := getMultisigTxOptions(account, subnetAuthKeys)
+	fxIDs := make([]ids.ID, 0)
+	subnetID, err := ids.FromString(params.SubnetID)
+	if err != nil {
+		return types.BuildTxResult{}, fmt.Errorf("failed to parse subnet ID: %w", err)
+	}
+	vmID, err := ids.FromString(params.VMID)
+	if err != nil {
+		return types.BuildTxResult{}, fmt.Errorf("failed to parse VM ID: %w", err)
+	}
+	unsignedTx, err := wallet.P().Builder().NewCreateChainTx(
+		subnetID,
+		params.Genesis,
+		vmID,
+		fxIDs,
+		params.ChainName,
+		options...,
+	)
+	if err != nil {
+		return types.BuildTxResult{}, fmt.Errorf("error building tx: %w", err)
+	}
+	builtTx := avagoTxs.Tx{Unsigned: unsignedTx}
+	pChainResult := types.NewPChainBuildTxResult(&builtTx)
+	return types.BuildTxResult{BuildTxOutput: pChainResult}, nil
+}
+
+// buildConvertSubnetToL1Tx provides a default implementation that can be used by any wallet
+func buildConvertSubnetToL1Tx(wallet *primary.Wallet, account account.Account, params *pchainTxs.ConvertSubnetToL1TxParams) (types.BuildTxResult, error) {
+	subnetAuthKeys, err := convertSubnetAuthKeys(params.SubnetAuthKeys)
```

Comment:
Why do we bother converting keys in this step? Should we instead just have params.SubnetAuthKeys be a slice of shortIDs?